app/components/traffic-measure/index.ts

```diff
@@ -195,14 +196,14 @@ export default class TrafficMeasureIndexComponent extends Component<Args> {
 
   @action
   async addSign(sign: TrafficSignConcept) {
-    this.signs.push(sign);
+    this.signs = [...this.signs, sign];
```

Comment:
Since `this.signs` is an ember array I think you can use `pushObject` here. At the moment you lose the ember array when adding (and removing) signs, so I think it will error again if you add a sign and then reorder them.